index.html

```diff
@@ -497,6 +497,27 @@ <h3>
           Details of how to actually get the [=digital credential=] are
           forthcoming.
         </aside>
+        <ol class="algorithm" >
+          <li>
+            <aside class="issue">
+              The following guidance on handling multiple requests will need to be relocated once
+              this entire section of the algorithm is defined.
+            </aside>
+            When multiple [=digital credential/presentation requests=] are present
+            in {{DigitalCredentialRequestOptions/requests}},
+            [=user agents=] MUST only return one
+            [=digital credential/presentation responses | presentation response=] for one
+            of these requests.
+            Multiple credentials MAY be present in this one response depending on the capabilities
+            of the corresponding [=digital credential/exchange protocol=].
+            <aside class="note">
```

Comment:
Notes shouldn't include RFC2119 keywords. 

I don't think this even makes sense, to be honest. How they are presented is up to the user agent, not up to verifier.

Reply:
If we imagine a Verifier that is trying to handle a wallet ecosystem that's in between foo-proto-v1 and foo-proto-v2, how might they express a preference for getting a foo-proto-v2 response when a wallet supports it? I'm suggesting here that we approach it by allowing Verifiers to sorting the list of requests by protocol, from most preferred to least preferred.

Reply:
You can't express that preference, because that would allow targeting particular wallets. The way to express it is to just pass the request either exclusively for "-v2" or pass all of them... but it's up to the user to pick the wallet and credential they want. 

For example: I, as a user, have "Super Awesome Wallet" installed (its my favorite app) - but only supports V1... I, as user, don't care that the issuer wants "-v2", because "Super Awesome Wallet" has both my credential and provides such an awesome experience.

Reply:
Alright, I see why this can't be in here. I'll remove it.